env0.yml

```diff
@@ -0,0 +1,15 @@
+version: 1
+
+deploy:
+  steps:
+    setupVariables:
+      after: &login-k8s
+        - aws eks --region us-west-2 update-kubeconfig --name k8s-template-cluster-eksCluster-e8a88b9
```

Comment:
looks like it related to us only.
so please put it under
https://github.com/env0/integration-templates